### ISSUE TITLE
[Quest] Ogres droping Pure Un'Goro Samples

### ISSUE THREAD

**Issue by Xathal** [dalaranwow/dalaran-wow#4909]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Ogres in the Burning Steppes dropping Pure Un'Goro Samples

**Expected behaviour**: Using the experimentation set next to the quest giver in Undercity to procure Pure Un'Goro Samples. Ogres to be unrelated to quest.
![wowscrnshot_101417_204536](https://user-images.githubusercontent.com/32802233/31578818-4a79119e-b129-11e7-9f1c-a8f2a9be3f57.jpg)

**Steps to reproduce the problem**:

1. Acquire the quest "...and a Batch of Ooze"
2. (Presumably) collect Un'Goro Slime Samples
3. Ogres in Burning Steppes will drop Pure Un'Goro Samples

**Include proofs for this behavior**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

http://wowwiki.wikia.com/wiki/Quest:..._and_a_Batch_of_Ooze
http://www.wowhead.com/npc=7035/firegut-brute
http://www.wowhead.com/npc=7034/firegut-ogre-mage

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Thank you @Xathal for the report.
Confirmed on ptr.

**Comment by Rushor** [comment]
The issue has been fixed by a developer and will be tested on the Public Test Realm in the future (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm in the future.
This means that the issue exists in game at the moment i wrote this message.
**Only once the issue was marked with "Fixed - On Live Server", you can find the fix on your game server.**
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TeridanDW** [comment]
Works as intended on PTR.
